pkg/http2/server.go

```diff
@@ -25,7 +25,12 @@ import (
 	tConfig "github.com/dubbogo/triple/pkg/http2/config"
 )
 
-type Http2Handler func(path string, header http.Header, recvChan chan *bytes.Buffer, sendChan chan *bytes.Buffer, ctrlch chan http.Header, errCh chan interface{})
+// Http2Handler uses to relay data to upper layer and receive data from upper layer as well.
```

Comment:
Http2Handler relays data to upper layer and receives data from upper layer as well.